system/pages/serverinfo.php

```diff
@@ -56,22 +56,22 @@
 if(isset($whiteSkullTime))
     $whiteSkullTime = eval('return ' . $whiteSkullTime . ';');
 
-$redSkullLength = isset($config['lua']['redSkullLength']) ? $config['lua']['redSkullLength'] : NULL;
+$redSkullLength = isset($config['lua']['redSkullDuration']) ? $config['lua']['redSkullDuration'] : NULL;
 if(!isset($redSkullLength) && isset($config['lua']['red_skull_duration']))
     $redSkullLength = $config['lua']['red_skull_duration'];
 
 if(isset($redSkullLength))
     $redSkullLength = eval('return ' . $redSkullLength . ';');
 
-$blackSkull = false;
-$blackSkullLength = NULL;
+$blackSkull = true;
```

Comment:
deixe essa linha como false, se não, sempre será true.
```suggestion
$blackSkull = false;
```